spec/models/user_spec.rb

```diff
@@ -1785,4 +1785,27 @@ def create_user(options = {})
 
   end
 
+  describe '#show_profile_photo?' do
+    subject { user.show_profile_photo? }
+
+    context 'with a profile_photo' do
+      let(:user) { FactoryBot.create(:user) }
+
+      before do
+        user.create_profile_photo!(data: load_file_fixture('parrot.png'))
+      end
+
+      it { is_expected.to be_truthy }
+    end
+
+    context 'without a profile_photo' do
+      let(:user) { FactoryBot.build(:user) }
+      it { is_expected.to be_falsey }
+    end
+
+    context 'when banned' do
+      let(:user) { FactoryBot.build(:user, :banned) }
+      it { is_expected.to be_falsey }
```

Comment:
This spec could be passing because the banned user doesn't have a profile photo. We should add the before block used in the 'with a profile_photo' context.